public/database.php

```diff
@@ -8,7 +8,7 @@ function get_db_handle() : PDO {
     return $dbh;
   }
 
-  $dbh = new PDO("mysql:host=127.0.0.1;dbname=miniboard", MB_DB_USER, MB_DB_PASS, [
+  $dbh = new PDO("mysql:host=mariadb;dbname=miniboard", MB_DB_USER, MB_DB_PASS, [
```

Comment:
Since now there are two ways to develop the software (either running PHP from host or with this new option inside a container), shouldn't these variables be configurable via environment variables?

Reply:
With the app dockerized, you don't need to run the php app on host machine. This is good because you have the php app running in the same OS instead of multiple different systems across developers.

Reply:
Well I guess we can make the database connection more configurable in the future outside of this PR.